client/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "baseUrl": "./",
         "declaration": false,
         "downlevelIteration": true,
-        "emitDecoratorMetadata": true,
+        "emitDecoratorMetadata": false,
```

Comment:
Why did you change this?

Reply:
I came across the same error as in #1174 and read that this might fix it, since setting this option to `true` is no longer needed since angular 8. But it didn't help, so I'll revert it for now.